src/themes/default/elements/icon.overrides

```diff
@@ -201,6 +202,19 @@ i.icon.other.gender.horizontal:before { content: "\f22b"; }
 i.icon.neuter:before { content: "\f22c"; }
 i.icon.genderless:before { content: "\f22d"; }
 
+/* Accessibility */
+i.icon.universal.access:before { content: "\f29a"; }
+i.icon.alternative.wheelchair:before { content: "\f29b"; }
```

Comment:
New icon: `alternative wheelchair`, the proposal for a new iconography for access, by The Accessible Icon Project. **This should be renamed.**